src/demo/apps/ChartsApp.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
-import * as fdc3 from '../../client/index';
+import * as fdc3 from '../../client/main';
 
 import '../../../res/demo/css/w3.css';
 
 import { Chart } from '../components/charts/Chart';
-import { SecurityPayload, Payload } from '../../client/context';
+import { SecurityContext, Context } from '../../client/context';
```

Comment:
Spaces inside {} - are we worrying about linting for now?

Reply:
Current versions of GTS don't work properly with tsx files. Will need to be dealt with, but not part of this story.

Reply:
SERVICE-365